README.md

```diff
@@ -119,22 +122,36 @@ foreach ($authorizations as $authorization) {
 
 ### Self test
 
-After exposing the challenges (made accessible through HTTP or DNS) we can perform a self test just to 
-be sure it works. For a DNS test call:
-
-```php
-$client->selfTest($authorization, Client::VALIDATON_DNS);
-``` 
+After exposing the challenges (made accessible through HTTP or DNS) we should perform a self test just to 
+be sure it works before asking Let's Encrypt to validate ownership.
 
 For a HTTP challenge test call:
 ```php
-$client->selfTest($authorization, Client::VALIDATION_HTTP);
+if (!$client->selfTest($authorization, Client::VALIDATION_HTTP)) {
+    throw new \Exception('Count not verify ownership via HTTP');
+}
 ```
 
+For a DNS test call:
+
+```php
+if (!$client->selfTest($authorization, Client::VALIDATON_DNS)) {
+    throw new \Exception('Count not verify ownership via DNS');
+}
+sleep(30); // this further sleep is recommended, depending on your DNS provider, see below
+``` 
+
+With DNS validation, after the `selfTest` has confirmed that DNS has been updated, it is 
+recommend you wait some additional time before proceeding, e.g. `sleep(30);`. This is
```

Comment:
it is recommend**ed**

Reply:
Thanks for fixing my typos, great additions. This will definitely help devs.

Reply:
corrected :)